geoh5py/data/data.py

```diff
@@ -148,6 +149,11 @@ def copy_from_extent(
             **kwargs,
         )
 
+    @classmethod
+    def default_type_uid(cls) -> uuid.UUID | None:
```

Comment:
Can this be None?

Reply:
Yes, Data doesn't have default except for Geometry type